client/state/data-layer/wpcom/jetpack/settings/index.js

```diff
@@ -101,7 +108,7 @@ export const saveJetpackSettings = ( { dispatch }, action ) => {
 				path: '/jetpack-blogs/' + siteId + '/rest-api/',
 				body: {
 					path: '/jetpack/v4/settings/',
-					body: JSON.stringify( settings ),
+					body: JSON.stringify( toApi( settings ) ),
```

Comment:
Note that we don't seem to be testing this. Whether we have `toApi` or not should matter, right? Some tests could verify this.